src/Spryker/Zed/AppMerchant/Communication/Plugin/AppWebhook/AccountUpdatedWebhookHandlerPlugin.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Zed\AppMerchant\Communication\Plugin\AppWebhook;
+
+use Generated\Shared\Transfer\WebhookRequestTransfer;
+use Generated\Shared\Transfer\WebhookResponseTransfer;
+use Spryker\Zed\AppWebhook\Dependency\Plugin\WebhookHandlerPluginInterface;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+
+/**
+ * {@inheritDoc}
+ *
+ * @api
+ *
+ * @method \Spryker\Zed\AppMerchant\AppMerchantConfig getConfig()
+ * @method \Spryker\Zed\AppMerchant\Business\AppMerchantFacadeInterface getFacade()
+ */
+class AccountUpdatedWebhookHandlerPlugin extends AbstractPlugin implements WebhookHandlerPluginInterface
+{
+    public function canHandle(WebhookRequestTransfer $webhookRequestTransfer): bool
+    {
+        return true; // TODO Needs implementation
```

Comment:
```suggestion
        return isset($webhookRequestTransfer->getSomeWebhookPayload());
```
what can be checked here?

Reply:
It is fully implemented in my current task. Adding it from there would cause merge conflicts let's keep  here, will be fixed in the next one.